keyboards/mwstudio/cover65/keymaps/via/rules.mk

```diff
@@ -0,0 +1,3 @@
+VIA_ENABLE = yes
+LTO_ENABLE = yes
```

Comment:
Just a heads up, but you likely don't need LTO here, as it's mostly a space saving measure.